app/assets/stylesheets/helpers/_header.scss

```diff
@@ -27,20 +30,22 @@
     display: none;
 
     @media screen and (max-width: 379px) {
+      background-color: $govuk-brand-colour;
+      background-image: image-url("search-button.png");
```

Comment:
Is it possible to use the [search icon from the components gem](https://github.com/alphagov/govuk_publishing_components/blob/master/app/assets/images/govuk_publishing_components/search-button.png)?

Reply:
It is possible - I tried, got bogged down in quite a lot of strange side effects, reverted the commit, and decided to just stick to colours for this pull request. Components are next!